features/0092-transport-return-route/README.md

```diff
@@ -0,0 +1,88 @@
+# 0092: Transports Return Route
+- Author: Sam Curren <sam@sovrin.org>
+- Start Date: 2019-03-04
+
+## Status
+- Status: [PROPOSED](/README.md#rfc-lifecycle)
+- Status Date: 2019-06-18
+- Status Note: Supersedes [INDY HIPE PR 116](<https://github.com/hyperledger/indy-hipe/pull/116>)
```

Comment:
@TelegramSam The <> pattern is a markdown hyperlink short-cut. The [...](...) form is the long form. Combining them is unnecessary and will break the the hyperlink unit test. Can you please remove the <>?

Reply:
(I will update the unit test to tolerate this, though.)